packages/edit-site/src/components/sidebar-edit-mode/settings-header/index.js

```diff
@@ -23,10 +23,10 @@ import { POST_TYPE_LABELS, TEMPLATE_POST_TYPE } from '../../../utils/constants';
 const SettingsHeader = ( { sidebarName } ) => {
 	const { isEditingPage, entityType } = useSelect( ( select ) => {
 		const { getEditedPostType, isPage } = select( editSiteStore );
-		const { getRenderingMode } = select( editorStore );
+		const { getCurrentPostType } = select( editorStore );
 
 		return {
-			isEditingPage: isPage() && getRenderingMode() !== 'template-only',
+			isEditingPage: isPage() && getCurrentPostType() !== 'wp_template',
```

Comment:
Feels like we should get rid of this boolean and always rely on the post type label.

Reply:
Yeah, agree, seems like it would have the same result!

Looking into it, I'm not sure I fully understand why the labels are stored in a constant `POST_TYPE_LABELS` vs grabbing them from the post type entity 🤔

Reply:
Seems like it's from [Site editor: use constants rather than hard coded template strings (round 3)](https://github.com/WordPress/gutenberg/pull/54705#top).

cc @ramonjd

Reply:
The `POST_TYPE_LABELS` constant was introduce here rather: https://github.com/WordPress/gutenberg/pull/54484/files#diff-53f39da7a5f1f996f5359a2dd7b2747fb3d60a494523dd73dba946fab8e82592L174

I seem to recall it was just an centralisation of what was already happening.

So we were using `typeLabels[ editedPostType ]` internal to a few files, and those translations were required elsewhere.

Feel free to change as you see fit 👍🏻

Reply:
Looks like the labels were originally introduced in this PR for the document title - [Site Editor: Append template type and name to the site editor page title](https://github.com/WordPress/gutenberg/pull/47855#top).

I think maybe they're being used for things they shouldn't be used for now. Most of the time the post type entity's labels will satisfy needs.

Reply:
Ok, I've changed it to use the `getPostTypeLabel` selector.

The one visible difference seems to be that when editing a navigation menu the label shows 'Navigation Menu' instead of just 'Navigation'. 🤔 

Might be ok, or maybe it's considered one special case.